extension/background/youtubei.js

```diff
@@ -54,3 +60,109 @@ function parseTimestamp(ts) {
     const hours = parseInt(parts[2]) || 0
     return secs + (60 * mins) + (60 * 60 * hours)
 }
+
+// time-comments related code below
+
+const INNERTUBE_API_KEY = "AIzaSyAO_FJ2SlqU8Q4STEHLGCilw_Y9_11qcW8"
+const INNERTUBE_CLIENT_NAME = "WEB"
+
+async function fetchChaptersFromComments(videoResponse) {
```

Comment:
It doesn't seems like a part of `youtubei` API
Could we use this function to `background.js`?

Reply:
How about moving this function, say, to comments.js?

Reply:
It's not necessary IMO. `background.js` isn't that big.
We have `youtubei.js` for YouTube API and `background.js` for business logic.

Reply:
Okay.